src/main/java/com/amazon/dlic/auth/http/jwt/AbstractHTTPJwtAuthenticator.java

```diff
@@ -56,16 +56,20 @@ public abstract class AbstractHTTPJwtAuthenticator implements HTTPAuthenticator
     private final String subjectKey;
     private final String rolesKey;
 
+    public static final int DEFAULT_CLOCKSKEWTOLERANCESECONDS = 300;
```

Comment:
Please upper snake case
```suggestion
    public static final int DEFAULT_CLOCK_SKEW_TOLERANCE_SECONDS = 300;
```